modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-cli_{context}"]
+= Installing the {TempoShortName} by using the CLI
+
+You can install the {TempoShortName} on the command line.
+
+.Prerequisites
+
+* An active OpenShift CLI (`oc`) session by a xref:https://docs.openshift.com/container-platform/latest/admin_solutions/user_role_mgmt.html#creating-a-cluster-administrator[cluster administrator with the `cluster-admin` role]. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
++
+TIP: Ensure that your OpenShift CLI (`oc`) version is up to date and matches your {product-title} version.
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+. Install the {TempoOperator}:
+
+.. Create a subscription.
++
+[source,yaml]
+----
+$ oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: tempo-product
+  namespace: openshift-operators
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: tempo-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.. Check the Operator status.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-operators
+----
+
+. Run `oc apply -f` to create a Secret for your object storage bucket.
++
+[TIP]
+====
+[source,terminal]
+----
+$ oc apply -f <secret_file>.yaml
+----
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
+====
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Use `oc create namespace` or `oc apply` to create a namespace for the *TempoStack* instance that you will create in the next step.
++
+[TIP]
+====
+[source,terminal]
+----
+$ oc create namespace <namespace_value_of_your_choice>
+----
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
```

Comment:
Remove this config to apply monitoring labels for the namespace in which TempoStack is created. This labels are needed only in the namespace in which Tempo Operator is installed and not in the namespace in which TempoStack is created, for TempoStack namespace, we use user workload monitoring.

Reply:
Done. Thank you for the explanation.